### ISSUE TITLE
Details about DGCNN

### ISSUE THREAD

**Issue by ham-stan** [lulutang0608/Point-BERT#42]
In your code, it seems that the graph is not recomputed in the feature space in each layer. So a static graph is yielded?

**Comment by yuxumin** [comment]
Yes, we use the original coordinates for graph computing in each layer.
